README.md

```diff
@@ -238,7 +232,14 @@ From there you can select targets to build and debug.
 
 We have set up the repo for easy integration into other projects which don't use CMake. Two main features make this possible:
 
-- To resolve all header file relative paths, you only need to include `sdk/inc` in your project. All header files are included in the sdk with relative paths to clearly demarcate the services they belong to. A couple examples being:
+- To resolve all header file relative paths, you only need to include `sdk/inc` in your project. All available header files for a service can be included with our simplified, all-inclusive service headers (`<az/az_<service>.h>`). For example:
+
+```c
+#include <az/az_core.h>
+#include <az/az_iot.h>
+```
+
+If you want to be more particular about what you include, you may include individual files as demonstrated here:
```

Comment:
Nit: more `specific` instead of `particular`.

Also, I think it's implicit that devs can specify multiple include files. There isn't any new information - we can remove the alternative.